### ISSUE TITLE
Persistent window sizes (using `sizehint` rather than `size`?)

### ISSUE THREAD

**Issue by kbarros** [JuliaPlots/Makie.jl#3767]
When executing a sequence of Makie plotting commands, the existing window will be reused, with a persistent position (great!). In this case, I would propose that the size of the existing window should also be persistent by default.

Interface brainstorming:

- One option is to reinterpret the `size` keyword to actually mean "size hint". That is, `size` would be ignored if a window already exists. Users can instead call `resize!` to force resizing.
- Another option is to preserve the meaning of the `size` keyword, but add a second keyword option `sizehint` option, and make the latter the default for all the plotting functions.

**Comment by jkrumbiegel** [comment]
Ah I think I understand now what workflow you want. Maybe GLMakie could use an additional screen config option called `resize_to_screen` or so which you could set to true

**Comment by kbarros** [comment]
Hi, please let me clarify. The workflow I'd like to achieve is illustrated in the following:

```julia
data = randn(5)
scatter(data)

# << user manually resizes the window by dragging the corner >>

scatter(data) # <-- I want to keep the same window position/size here
```

**Comment by jkrumbiegel** [comment]
yes, that's why I think an option in GLMakie itself makes more sense than one per-plot. As you want to write `scatter(data)` and not `scatter(data; some_other_option_whatever_its_called)`

**Comment by SimonDanisch** [comment]
Now that you say this, WGLMakie does actually have `resize_to=:parent/body`, so I guess it could be an option in GLMakie too, set by `GLMakie.activate!(resize_to=:window)` (maybe).
